snuba/datasets/querylog.py

```diff
@@ -0,0 +1,89 @@
+from datetime import timedelta
+from snuba import settings
+from snuba.clickhouse.columns import (
+    Array,
+    ColumnSet,
+    DateTime,
+    Enum,
+    Float,
+    LowCardinality,
+    Mapping,
+    Nested,
+    String,
+    UInt,
+)
+
+from snuba.datasets.dataset import TimeSeriesDataset
+from snuba.datasets.dataset_schemas import DatasetSchemas
+from snuba.datasets.schemas.tables import MergeTreeSchema
+from snuba.datasets.table_storage import TableWriter, KafkaStreamLoader
+from snuba.datasets.querylog_processor import QuerylogProcessor
+from snuba.query.extensions import QueryExtension
+from snuba.query.project_extension import ProjectExtension, ProjectExtensionProcessor
+from snuba.query.timeseries import TimeSeriesExtension
+
+
+class QuerylogDataset(TimeSeriesDataset):
+    def __init__(self) -> None:
+        status_type = Enum([("success", 0), ("error", 1), ("rate-limited", 2)])
+
+        columns = ColumnSet(
+            [
+                ("request", String()),
+                ("referrer", LowCardinality(String())),
+                ("dataset", LowCardinality(String())),
+                ("projects", Array(UInt(64))),
```

Comment:
Do you want the organization if available?

Reply:
Yeah, will add the field (even if not yet populated)